serverless/deploy_cpu.sh

```diff
@@ -4,19 +4,23 @@
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 FUNCTIONS_DIR=${1:-$SCRIPT_DIR}
 
-nuctl create project cvat
+docker build -t cvat.openvino.base "$SCRIPT_DIR/openvino/base"
```

Comment:
In our documentation we have some lines to build nuclio functions without ``deploy_cpu.sh`` script.

https://opencv.github.io/cvat/docs/contributing/setup-additional-components/

Don't they work anymore?